CPP_STYLE_GUIDE.md

```diff
@@ -168,7 +168,7 @@ What it says in the title.
 
 ## Ownership and Smart Pointers
 
-"Smart" pointers are classes that act like pointers, e.g.
+"Smart" pointers are classes that act like pointers, for example:
```

Comment:
The `:` is fine for the parenthetical stuff but it's a problem in ordinary prose like this. I don't think we want to do a global find/replace on this. We want to evaluate each item independently. For example, this instance should probably be changed to `such as` rather than `for example:`. 

Nit: Get rid of the scare quotes in this sentence and instead italicize the term being defined:

`"Smart" pointers` -> `_Smart pointers_`

Reply:
@Trott Done. Please review.